packages/cli/src/utils/getContractArtifact.ts

```diff
@@ -11,18 +11,24 @@ export type GetContractArtifactResult = {
   deployedBytecodeSize: number;
 };
 
+function HexNotStrict(value: string): value is Hex {
```

Comment:
```suggestion
function isBytecode(value: string): value is Hex {
```

Reply:
there's a better version of this in `world/ts/node/types.ts` if we wanna export that and reuse it (or lift it into common package0

Reply:
Right! Saw that but wanted to keep the changes as small as possible,  but happy to include that here if you think that's better

Reply:
for now will keep this so we can refactor and unify using the arktype one in a separate PR